commit 66094bebac80fe239de82e5b2dd6ad86c1bf7d0a (refs/changes/96/4496/2)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-08-02T15:31:47+00:00 (1 year, 2 months ago)
    
    joyent/node-workflow#163 package.json repository does not point to Joyent
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/package.json b/package.json
index 7544fa7..7010cf5 100644
--- a/package.json
+++ b/package.json
@@ -4,7 +4,7 @@
     "version": "1.2.0",
     "repository": {
         "type": "git",
-        "url": "git://github.com/kusor/node-workflow.git"
+        "url": "git://github.com/joyent/node-workflow.git"
     },
     "author": "Pedro Palazón Candel <kusorbox@gmail.com> (http://www.joyent.com)",
     "license": "MIT",
